File: Algoritmo/ProjetoJava/src/ExerciciosPDF/ex12.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ExerciciosPDF;

import javax.swing.JOptionPane;

/**
 *
 * @author vitor
 */
public class ex12 {
    public static void main(String[] args) {
        
        double velocidade;
        
        JOptionPane.showMessageDialog(null, "Transformação de m/s para km/h");
        
        // velocidade fornecida
        velocidade = Float.parseFloat(JOptionPane.showInputDialog("Digite uma velocidade: "));
        
        // multiplicando o valor em m/s por 3,6
        double velocidadeFinal = velocidade * 3.6;
        
        // velocidade em Km/h
        System.out.println("Velocidade em m/s convertida para " +velocidadeFinal+ "Km/h");
    }
}

File: Algoritmo/ProjetoPython/ex12.py
# Faça um algoritmo que transforme uma velocidade fornecida em m/s pelo usuário para Km/h. Para tal, multiplique o
# valor em m/s por 3,6.

print('*'*40)
print(' Transformação de m/s para km/h')
print('*'*40)

# velocidade fornecida
velocidade = float(input('Digite uma velocidade: '))

# multiplicando o valor em m/s por 3,6
velocidadeFinal = velocidade * 3.6

print('-'*40)
# velocidade em Km/h
print(f'Velocidade em {velocidade}m/s convertida para {velocidadeFinal}Km/h')
